src/App.css

```diff
@@ -0,0 +1,91 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    background-color: #ffc74a;
+}
+header {
+     height: auto;
+     width: auto;
+     margin-bottom: 30px;
+ }
+
+.logo{  
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    height:auto;
+    width:25%;
+    position: relative;
+}
+.pageButtons{
```

Comment:
CSS klassen mogen geen CamelCasing bevatten, dat doen we alleen bij JavaScript. Dus `page-buttons`